xml/System.Net.Security/NegotiateAuthentication.xml

```diff
@@ -508,13 +518,23 @@ For the client side of the authentication, the property returns the target name
         <Parameter Name="isEncrypted" Type="System.Boolean" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="input">To be added.</param>
-        <param name="outputWriter">To be added.</param>
-        <param name="requestEncryption">To be added.</param>
-        <param name="isEncrypted">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="input">Input message to be wrapped.</param>
+        <param name="outputWriter">Buffer writter where the wrapped message is written.</param>
+        <param name="requestEncryption">Specifies whether encryption is requested.</param>
```

Comment:
```suggestion
        <param name="requestEncryption">
          <see langword="true" /> to request encryption; otherwise, <see langword="false" />.</param>
```